Tests/SwiftParserTest/Assertions.swift

```diff
@@ -12,7 +12,7 @@
 
 import XCTest
 @_spi(RawSyntax) import SwiftSyntax
-@_spi(Testing)@_spi(RawSyntax) import SwiftParser
+@_spi(Testing) @_spi(RawSyntax) import SwiftParser
```

Comment:
I ran format.py before opening the PR, and it made this change 🤔 
Can I include this in this PR?

Reply:
Ah, that’s because the 5.9 `swift-format` doesn’t add a space here. It shouldn’t be included in the PR or CI will complain that the `release/5.9` branch is not formatted correctly according to the `release/5.9` branch of `swift-format`.